00_Base/src/ocpp/model/enums/index.ts

```diff
@@ -188,7 +188,8 @@ export const enum DataEnumType {
   boolean = "boolean",
   OptionList = "OptionList",
   SequenceList = "SequenceList",
-  MemberList = "MemberList"
+  MemberList = "MemberList",
+  passwordString = "passwordString"
```

Comment:
@thanaParis This file is generated and might be overwritten when the toolchain is ran again, do we know why this was not in the OCPP 2.0.1 enums as per specification?

Reply:
As far as I know it's an oversight. They are also missing identifierString, which is fairly important as it enforces a validation (charging station ids CANNOT have ':' in them) that breaks the protocol if not followed.